src/modules/menu/components/LayerCatalogueItem.vue

```diff
@@ -67,7 +67,7 @@ const activeLayers = computed(() => store.state.layers.activeLayers)
 const openThemesIds = computed(() => store.state.topics.openedTreeThemesIds)
 
 const hasChildren = computed(() => item.value?.layers?.length > 0)
-const hasLegend = computed(() => canBeAddedToTheMap.value && item.value?.hasLegend)
+const hasLegend = computed(() => canBeAddedToTheMap.value && item.value?.hasDescription)
```

Comment:
@pakb is hasDescription a good variable name? In that case I should probably update all the variable names which referer to legend instead of description.

Reply:
I can't think of a better name if we want to keep it short... So let's go with that :wink: